packages/core/lib/src/data/text_style.dart

```diff
@@ -108,33 +108,33 @@ class TextStyleHtml {
 /// A text styling builder.
 class TextStyleBuilder<T1> {
   /// The parent builder.
-  final TextStyleBuilder parent;
+  final TextStyleBuilder? parent;
 
-  List<Function> _builders;
-  List _inputs;
-  TextStyleHtml _parentOutput;
-  TextStyleHtml _output;
+  List<Function>? _builders;
+  List? _inputs;
+  TextStyleHtml? _parentOutput;
+  TextStyleHtml? _output;
 
   /// Create a builder.
   TextStyleBuilder({this.parent});
 
   /// Enqueues a callback.
   void enqueue<T2>(
-    TextStyleHtml Function(TextStyleHtml tsh, T2 input) builder, [
-    T2 input,
+    TextStyleHtml? Function(TextStyleHtml? tsh, T2 input)? builder, [
```

Comment:
```suggestion
    TextStyleHtml Function(TextStyleHtml tsh, T2? input) builder, [
```

Reply:
Changed builder and the return value to be not nullable. Was it intentional to make tsh non-nullable and input nullable? That seems to cause test failures, may need more investigation

Reply:
Yes, there are some builder that doesn't need any input. The `input` param of `enqueue` is optional itself.

Reply:
Then I will make that change as well, thank you for clarifying :)

Reply:
I did somthing a bit different: 
The function signature is now: 
```dart
TextStyleBuilder enqueueTsb<T>(
    TextStyleHtml Function(TextStyleHtml tsh, T input) builder,
    T input,
  )
```
This enforces that the type of `input` in the builder is the same as `input` of enqueue. Note that `builder` and `input` of enqueue can no longer be optional, enqueueing a builder without input would look like this: (You have to manually pass null as the input)
```dart
enqueueTsb((textStyleHtml tsh, Null input){}, null);
```

I renamed the function to `enqueTsb` and made `TextStyleBuilder tsb` public to be able to access it without registering a builder.

Reply:
Can we drop `enqueueTsb()` and let caller do `tsb.enqueue()` directly? With null safety, `tsb` is guaranteed to be not null anyway?

Reply:
Sure, that makes more sense. Done!